src/fu-tool.c

```diff
@@ -1301,6 +1301,68 @@ fu_util_detach (FuUtilPrivate *priv, gchar **values, GError **error)
 	return fu_device_detach (device, error);
 }
 
+static gboolean
+fu_util_unbind_driver (FuUtilPrivate *priv, gchar **values, GError **error)
+{
+	g_autoptr(FuDevice) device = NULL;
+	g_autoptr(FuDeviceLocker) locker = NULL;
+
+	/* load engine */
+	if (!fu_util_start_engine (priv, FU_ENGINE_LOAD_FLAG_NONE, error))
+		return FALSE;
+
+	/* get device */
+	if (g_strv_length (values) == 1) {
+		device = fu_util_get_device (priv, values[0], error);
+		if (device == NULL)
+			return FALSE;
+	} else {
+		device = fu_util_prompt_for_device (priv, NULL, error);
+		if (device == NULL)
+			return FALSE;
+	}
+
+	/* run vfunc */
+	locker = fu_device_locker_new (device, error);
+	if (locker == NULL)
+		return FALSE;
+	return fu_device_unbind_driver (device, error);
+}
+
+static gboolean
+fu_util_bind_driver (FuUtilPrivate *priv, gchar **values, GError **error)
+{
+	g_autoptr(FuDevice) device = NULL;
+	g_autoptr(FuDeviceLocker) locker = NULL;
+
+	/* load engine */
+	if (!fu_util_start_engine (priv, FU_ENGINE_LOAD_FLAG_NONE, error))
+		return FALSE;
+
```

Comment:
Since this is user facing I think it's worth adding some magic here to check if the driver provided exists in the kernel, and/or if `-` or `_` are done right.

For example for me I can interact with the kernel module `i2c-hid`, but `/sys/module/i2c_hid` is what you interact with.
So you probably need to check first if `/sys/module/%s` exists and if it doesn't, when you `s/-/_` if it does.

And I can totally see this magic test happening in `fu_device_bind_driver` too, but I think it will make this more user friendly.

Reply:
> s/-/_ if it does

I think modprobe always unconditionally does this, maybe we just do the same?

Reply:
Sure, makes sense to me.

Reply:
Done.